added new applier and moved exceptions into seperate file
--This line, and those below, will be ignored--

M    rios/imageio.py
M    rios/inputcollection.py
A    rios/applier.py
M    rios/imagewriter.py
M    rios/calcstats.py
A    rios/exceptions.py
M    rios/imagereader.py
